workbench/icons/Makefile.am

```diff
@@ -0,0 +1,20 @@
+icondir = $(datadir)/icons/hicolor/16x16/apps
```

Comment:
How do I do that? Just append more icon files to the makefile. How are the icons selected by the application? (The variable ```icondir``` doesn't seem to be used by the makefile???)